github/ci/prow-deploy/files/jobs/kubevirt/containerized-data-importer/containerized-data-importer-postsubmits.yaml

```diff
@@ -195,3 +195,45 @@ postsubmits:
         resources:
           requests:
             memory: "8Gi"
+  - name: push-cdi-goveralls
+    cluster: kubevirt-prow-control-plane
+    annotations:
+      fork-per-release: "true"
```

Comment:
Also we probably don't want the `fork-per-release` annotation here either.

Reply:
I mostly copy-pasted from the kubevirt/kubevirt run. Is there anywhere I can read about all these options?

Reply:
Yes sure - the prow documentation is pretty good for explaining each of the fields: 

https://docs.prow.k8s.io/docs/jobs/#how-to-configure-new-jobs

https://github.com/kubernetes/test-infra/blob/master/testgrid/config.md